src/custom.scss

```diff
@@ -0,0 +1,5 @@
+// We can override bootstrap variables by setting them before the import
+$body-bg: #000;
+
+// Import Bootstrap and its default variables
+@import '~bootstrap/scss/bootstrap.scss';
```

Comment:
I would rather see all this code in `index.css` since it is the top-level SCSS file.

Also, we could probably rename `index.css` to `index.scss`.